src/registrar/tests/test_transition_domain_migrations.py

```diff
@@ -19,6 +19,233 @@
 from .common import less_console_noise
 
 
+class TestOrganizationMigration(TestCase):
+    def setUp(self):
+        """ """
+        # self.load_transition_domain_script = "load_transition_domain",
+        # self.transfer_script = "transfer_transition_domains_to_domains",
+        # self.master_script = "load_transition_domain",
+
+        self.test_data_file_location = "registrar/tests/data"
+        self.test_domain_contact_filename = "test_domain_contacts.txt"
+        self.test_contact_filename = "test_contacts.txt"
+        self.test_domain_status_filename = "test_domain_statuses.txt"
+
+        # Files for parsing additional TransitionDomain data
+        self.test_agency_adhoc_filename = "test_agency_adhoc.txt"
+        self.test_authority_adhoc_filename = "test_authority_adhoc.txt"
+        self.test_domain_additional = "test_domain_additional.txt"
+        self.test_domain_types_adhoc = "test_domain_types_adhoc.txt"
+        self.test_escrow_domains_daily = "test_escrow_domains_daily"
```

Comment:
**comment/non-blocking:** Given this script is just supposed to be a one off, this is red flag for me to see so much setup for a test that then requires other files to be set. I would think the script would just take one file only, as we are NOT going to re-run the data migration at all, so we do not need the other pieces of the migration scripts touched or expanded on.

Reply:
You are correct, those are not needed. Cleaned up that portion